docs/chat_setup/chat_setup.md

```diff
@@ -61,6 +61,24 @@ The plugin behavior can be controlled with the following list of general setting
 
 TODO: clarify the above statement, starting from "The plugin behavior can be controlled . . ."
 
+### Run Migrations
+
+Once the setup is done, the Nautobot `post_install` command needs to be run. As the _Nautobot user_ (which following Nautobot install docs is `nautobot`) execute:
```

Comment:
I would suggest changing "the Nautobot `post_install` command" to "the Nautobot `post_upgrade` command".